### DISCUSSION TITLE
New Angular Project 添加ng add ng-alain後畫面異常

### DISCUSSION THREAD

**Discussion by jul101** [ng-alain/ng-alain#2091] [Q&A]
小弟正在試用Alain 12的版本，碰到了添加Alain後畫面異常的狀況

最先是Login畫面輸入了沒反應，接著先暫時拔掉Login畫面進到Dashboard後沒有顯示正常的官網Demo首頁

![image](https://user-images.githubusercontent.com/9526204/132618026-e21a10ef-2a4d-4301-a1c7-2479bc1752c9.png)

colsole也沒有任何報錯，目前不知道朝哪方面確認問題

這是目前我的版本相關資訊，還請各位先進指點一下可能的問題，謝謝

Angular CLI: 12.2.5
Node: 12.22.6
Package Manager: npm 6.14.15
OS: win32 x64

Angular: 12.2.5
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1202.5
@angular-devkit/build-angular   12.2.5
@angular-devkit/core            12.2.5
@angular-devkit/schematics      12.2.5
@schematics/angular             12.2.5
rxjs                            6.6.7
typescript                      4.3.5

**Comment by cipchk** [comment]
看起来像是缺少 Css 的加载，应该可以在 Chrome 的 Network 面板中看出这一细节。

**Reply by jul101** [reply]
先謝謝您的回覆，確認過network的資訊，實在也看不出端倪

![image](https://user-images.githubusercontent.com/9526204/132625249-244aa2bb-a9e9-4707-96e0-415ac9b3fc1d.png)

**Reply by cipchk** [reply]
看起来一切正常，能否提供一下你创建 NG-ALAIN 的完整步骤，以及所有命令运行的日志吗？包括 `npm run start` 时所产生的所有日志。这有助于分析。

**Comment by jul101** [comment]
由於在家裡第一次安裝，所以從nvm install nodejs 開始進行的

步驟如下
nvm install 12.14
npm install -g typescript@4.2
npm install -g @angular/cli@12
ng new blank-project --style scss --routing
cd blank-project
ng add ng-alain@12

log的話是貼ng serve後的console log嗎?

我先貼看看，先謝謝您的回覆~
```
PS D:\Angular\blank-project> ng s --open
✔ Browser application bundle generation complete.

Initial Chunk Files                             | Names         |      Size    
vendor.js                                       | vendor        |   9.50 MB    
polyfills.js                                    | polyfills     | 510.60 kB    
styles.css, styles.js                           | styles        | 383.80 kB    
main.js                                         | main          | 221.72 kB    
runtime.js                                      | runtime       |  12.55 kB    

                                                | Initial Total |  10.60 MB    

Lazy Chunk Files                                | Names         |      Size    
src_app_routes_exception_exception_module_ts.js | -             |  30.22 kB    

Build at: 2021-09-09T04:48:45.256Z - Hash: 1d9021ed89ab55a2b982 - Time: 64604ms

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **


√ Compiled successfully.
✔ Browser application bundle generation complete.

6 unchanged chunks

Build at: 2021-09-09T04:48:56.377Z - Hash: 7e470bce450a9c19f0b3 - Time: 9483ms

√ Compiled successfully.
```

**Reply by cipchk** [reply]
我的建议是根据文档的[安装](https://ng-alain.com/docs/getting-started/zh#安装)节点的步骤做一下尝试。
